Extending/Property-Editors/value-converters.md

```diff
@@ -2,18 +2,18 @@
 
 **Applies to Umbraco 7 and newer**
 
-A property value converter converts a property editors database stored value to another type. The converted value can be accessed from MVC Razor or any other Published Content API. 
+A Property Value Converter converts a property editor's database-stored value to another type. The converted value can be accessed from MVC Razor or any other Published Content API. 
 
 Starting with Umbraco v7, published property values have four "Values":
 
-- Data - The raw data stored in the database, this is generally a `String`
-- Source - A object of type that is appropriate to the property, e.g. a nodeId should be a `Int` or a collection of nodeId's would be a array of `Int[]`
-- Object - The object to be used when accessing the property using a Published Content API e.g. UmbracoHelper `GetPropertyValue<T>` method
-- XPath - The object to be used when the property is accessed by XPath, this should generally be a `String` or a `XPathNodeIterator` 
+- **Data** - The raw data stored in the database, this is generally a `String`
+- **Source** - An object of a type that is appropriate to the property, e.g. a nodeId should be an `Int` or a collection of nodeIds would be an integer array, `Int[]`
+- **Object** - The object to be used when accessing the property using a Published Content API, e.g. UmbracoHelper's `GetPropertyValue<T>` method
+- **XPath** - The object to be used when the property is accessed by XPath; This should generally be a `String` or an `XPathNodeIterator`
```

Comment:
do we need a ;-sign?

Reply:
Could easily do with a period (**.**) instead, if the semicolon feels weird :-)